File: Web/RMA-Web-Shell/projects/digicare/src/app/medical-form-manager/models/final-medical-report-form.ts
import { MedicalReportForm } from 'projects/digicare/src/app/medical-form-manager/models/medical-report-form';

export class FinalMedicalReportForm {
  finalMedicalReportFormId: number;
  medicalReportForm: MedicalReportForm;
  mechanismOfInjury: string;
  injuryOrDiseaseDescription: string;
  additionalContributoryCauses: string;
  impairmentFindings: string;
  isStabilised: boolean;
  dateReturnToWork: Date;
  dateStabilised: Date;
  pevStabilisedDate: Date;
}

File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/FinalMedicalReportForm.cs
namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class FinalMedicalReportForm
    {
        public int FinalMedicalReportFormId { get; set; }
        public int MedicalReportFormId { get; set; }
        public string MechanismOfInjury { get; set; }
        public string injuryOrDiseaseDescription { get; set; }
        public string AdditionalContributoryCauses { get; set; }
        public string ImpairmentFindings { get; set; }
        public bool IsStabilised { get; set; }
        public System.DateTime? DateReturnToWork { get; set; }
        public System.DateTime? DateStabilised { get; set; }
        public System.DateTime? PevStabilisedDate { get; set; }
        public MedicalReportForm MedicalReportForm { get; set; }

    }
}
